### PR TITLE
Minor Adjustments

### PR DESCRIPTION
**Pull request by w0lfschild**
- load without trash tile state change
- subtract .DS_Store from count

### CODE REVIEW

File: binventory/binventory/binventory.m [hunk 1]
```diff
@@ -22,52 +22,21 @@ - (void)removeStatusLabelForType:(int)arg1;
...
-
-        OSErr	err	= FSGetCatalogInfo(&ref, kFSCatInfoValence, &catInfo, NULL, NULL, NULL);
-        if (err == noErr)
-            x += catInfo.valence;
-        
+    for (NSURL *url in Trashes)
+    {
+        if ([[NSFileManager defaultManager] fileExistsAtPath:[NSString stringWithFormat:@"%@/.DS_Store", url.path]])
+            x -= 1;
+        x += [[[NSFileManager defaultManager] contentsOfDirectoryAtPath:[url path] error:nil] count];
```
Comment (alexzielenski) [inline binventory/binventory/binventory.m]: This method is slower and not as accurate as the FS API which is why I didn't use it here. It's easier/faster to get the valence of the directory inode.

File: binventory/binventory/binventory.m [hunk 2]
```diff
@@ -22,52 +22,21 @@ - (void)removeStatusLabelForType:(int)arg1;
 @implementation DKTrashTile
 
 - (void)dk_updateCount {
-    NSUInteger x = 0;
+    long x = 0;
```
Comment (alexzielenski) [inline binventory/binventory/binventory.m]: Never use primitive types like this when writing code for Darwin platforms using Apple SDKs. You should always opt for NSInteger/NSUInteger when you don't care about the width of the type.

**Comment by alexzielenski** [comment]
I made some inline comments, if you could fix them in your fork and rebase/squash them, I'll pull.

**Comment by w0lfschild** [comment]
Done.

**Pull request merged**